server/views/index.ejs

```diff
@@ -0,0 +1,52 @@
+<doctype html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <title>Pathlete</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css" />
+    <link rel="stylesheet" href="dist/css/app.css">
+  </head>
+  <body ng-app="pathleteApp">
+    <!--[if lt IE 7]>
+      <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
+
+    <div class="container">
+      
+      <header class="header" ng-controller="ToolbarCtrl" ng-show="showing">
+        <ul class="nav nav-pills pull-right">
+          <li><a ng-href="/progress">Progress</a></li>
+          <li><a ng-href="/achievements">Achievements</a></li>
+          <li><a ng-href="/logout" target="_self">Logout</a></li>
```

Comment:
Hmm... are you pulling in everything from Rick's file? I'm pretty sure these should now be `ui-sref`.